src/__integ__/use-browser-with-scrollbars.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+
+// Workaround until scrollbars are generally shown in tests (AWSUI-59983)
+
+interface TestFunction {
+  (browser: WebdriverIO.Browser): Promise<void> | void;
+}
+
+const options = {
+  capabilities: {
+    'goog:chromeOptions': {
+      args: [
+        // Same array as in
+        // https://github.com/cloudscape-design/browser-test-tools/blob/4aaed9e410b13e05a7d5dbace17231776d250b97/src/browsers/capabilities.ts
+        // but without --hide-scrollbar.
```

Comment:
We would need to explicitly export it in the `exports` field of the `package.json`. Currently it is not importable (I actually tried to import it).

Still, this would be an option just like a flag would be an option, but in any of these two cases it would be an API change for a temporary workaround. The goal to just show scrollbars everywhere, without opt-out.

Reply:
Ah I had missed that it was already present in the other class. Thanks!